docs/controls/Line.md

```diff
@@ -193,6 +194,17 @@ Each line can have a style object in [Values](#values) level.
 | --------------- | -------- | ------- | ------------------------------------------------------ |
 | strokeDashArray | string   | "0"     | Applies stroke-dasharray CSS property to the SVG line. |
 
+##### LegendOptions
+
+Each line can have a legendOptions object in [Values](#values) level.
+
+###### Required
+
+| Property Name | Expected | Default | Description                  |
+| ------------- | -------- | ------- | ---------------------------- |
+| showShape     | boolean  | true    | Display Shape in the legend. |
```

Comment:
To maintain passivity we need to set this as `true` and showLine as `false`. Let consumers decide what they want to show/hide after that.

Reply:
Updated